internal/io/http/client.go

```diff
@@ -208,8 +218,63 @@ func (cc *ClientConf) InitConf(device string, props map[string]interface{}, with
 		}
 	}
 
-	tr := &http.Transport{
-		TLSClientConfig: tlscfg,
+	var tr *http.Transport
+	edgexCredentialFile := os.Getenv("EDGEX_CREDENTIALS")
```

Comment:
Use the env as switch will affect all http connections. We should add a parameter in the props instead.

Reply:
that is true. OpenZiti __should__ try to use an OpenZiti connection first, and then fall back to default http transport in that case. OpenZiti is only used if the url being sent matches a pattern indicating it should be intercepted. I'm happy to add a param to the props file if that's the best way to do it. I only did it this way, to get this particular feedback. I'll look to rework with the props file.